src/SUMA/SUMA_clippingPlanes.c

```diff
@@ -131,7 +134,9 @@ Boolean toggleClippingPlaneMode(SUMA_SurfaceViewer *sv, Widget w, int *locallySe
         if (SUMAg_CF->clippingPlaneVerbose && SUMAg_CF->clippingPlaneVerbosityLevel>1)
             fprintf(stderr, "### Darken inactive clip planes\n");
         darkenInactiveClipPlaneSquares(*locallySelectedPlane);
+        /*
         lightenActiveClipPlaneSquare(*locallySelectedPlane);
+        /**/
```

Comment:
```suggestion
        /*
        lightenActiveClipPlaneSquare(*locallySelectedPlane);
        */
```

Reply:
this has now been done in this commit: 80a2191